### PR TITLE
Multilang: formal pt-BR translations

### PR DESCRIPTION
**Pull request by AlexandreSato**

### CODE REVIEW

File: selfdrive/ui/translations/main_pt-BR.ts [hunk 1]
```diff
@@ -989,12 +989,12 @@ trabalho definido</translation>
...
         <location filename="../qt/offroad/settings.cc" line="242"/>
         <source>Git Branch</source>
-        <translation>Ramo Git</translation>
+        <translation>Git Branch</translation>
     </message>
     <message>
         <location filename="../qt/offroad/settings.cc" line="243"/>
         <source>Git Commit</source>
-        <translation>Commit Git</translation>
+        <translation>Último commit</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_pt-BR.ts:997]:
```suggestion
        <translation>Último Commit</translation>
```
?

**Comment by sshane** [comment]
Can you explain the reason for the changes? Is it minor grammar fixes, or more? Sounds like we should have a template for multilang PRs!

**Comment by AlexandreSato** [comment]
>

In Portuguese we have the main language styles: standard, formal and colloquial (slang).  For example: "Trocar" is "change" in short standard->slang and "Alterar" is the same "change" but in formal style.  In the next PR I will pay attention to the template language model.

**Comment by AlexandreSato** [comment]
> Can you explain the reason for the changes? Is it minor grammar fixes, or more? Sounds like we should have a template for multilang PRs!

"Idioma" also exists in Brazilian Portuguese and means exclusively human language, "Língua, linguagem" can mean programming language (Python, C, Java...)

**Pull request merged**